pkg/multicast/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package multicast contains various utility functions to work with IPv6
+// multicast
+package multicast
```

Comment:
One more comment: Should these new packages be owned by a particular group in the core team? Take a look at CODEOWNERS.

There's a natural tendency for anything vaguely related to the datapath to end up as part of @cilium/bpf so that's one option. If we don't extend CODEOWNERS then these will be owned by the janitor so future reviews in this area will be up to whoever is assigned to janitor that week. If there's few changes in future this isn't likely to be a problem but if you anticipate more changes coming it might be useful to add this to a codeowner team to ensure that someone with multicast background can provide the reviews you need.

Reply:
I think it's closer to datapath, so if ower is needed, @cilium/bpf makes sense.